### PR TITLE
Pull request to merge development branch with main one

### PR DESCRIPTION
**Pull request by Ghiftee**
Changelog:
- Main HTML skeleton
- Used grids along with the entire project to better place elements and make them reactive
- Responsiveness
- UX changes according to different screen sizes (1024px wide and 768px wide, which are the ones for mobile and tablet)
- Images were taken from the original site
- Some of the svg were taken from the original site and some others from the fontawesome.com site
- Font used is roboto

### CODE REVIEW

**Comment by mimipeshy** [comment]
### STATUS : CHANGES REQUIRED :red_circle: :recycle: 

Hey team @Ghiftee and @Greg0109 :wave: :wave: great job on working on this project. The page is similar to the original one. Keep it up :clap: :clap:  However to make it perfect, a few changes need to be made :

- [ ] Add more social media links for Author 2 in the README file

### Mobile view
- [ ] Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to seach icon) 
([link to page](https://thenextweb.com/))

Original | Clone
------------ | -------------
image | image

Notice the difference :question: 

- [ ] Latest news section data overlap over each other.

Original | Clone
------------ | -------------
image | image

Notice the difference :question: 

- [ ] Latest funding section overlaps the latest news section

Original | Clone
------------ | -------------
image | image 

Notice the difference :question: 

- [ ] The latest deals section data overlap with each other

Original | Clone
------------ | -------------
image | image

Notice the difference :question: 

- [ ] Social media icons in the footer section should be aligned at the center of the border-radius.

Original | Clone
------------ | -------------
image | image

Notice the difference :question: 

### Tablet view

- [ ] Footer section should be identical to the main page

Original | Clone
------------ | -------------
image | image

Notice the difference :question: 

### Desktop View
- [ ] It contains TWN logo at the top left corner of the row ([link to page](https://thenextweb.com/))

Original | Clone
------------ | -------------
image | image

Notice the difference :question: 
 
#### Implementation details

* [ ]  CSS styles should use [media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) for 2 breakpoints ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/breakpoints_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
  * No small break point as it is mobile-first and the base styles are for the mobile screen. ([Screenshot on mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
  * medium: min-width around 768px
  * large: min-width 1024px

That's all :smile: kindly submit another review when done.

_Feel free to reach out to me on slack if you have any questions or something is unclear, I will be glad to help_
Regards
Peris Ndanu

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
